MyClass2.cls

```diff
@@ -2,24 +2,24 @@ public class MyClass {
 
 	public static void foo1() {
 		for (Integer i=0; i<10; i++) {
+			
+			List<SObject> items = [Select Id, Name FROM account LIMIT 10];
```

Comment:
This data query is inside a loop. Please move the query outside the loop and put records....

Reply:
@glallopitzi ignore: comment here

Reply:
dwqs

Reply:
cqa

Reply:
ignore this

Reply:
GREP_ME

Reply:
@glallopitzi ignore: this is a comment

Reply:
@glallopitzi ignore: this is a fucing comment

Reply:
latest